microsoft-edge/webview2/howto/debug.md

```diff
@@ -111,16 +111,19 @@ In Visual Studio Code, complete the following actions to debug your code.
             "Path": "%path%;e:/path/to/your/application/location; "
         },
         "useWebView": true,
+        // The following two lines setup source path mapping
+        "url": "file:///${workspaceFolder}/path/to/your/toplevel/FOO.HTML",
+        "webRoot": "${workspaceFolder}/path/to/your/assets"
     ```  
-    
+        > [!NOTE]
+    > Because Visual Studio Code source path mapping requires the URL configuration setup your application will receive a command line parameter on startup. If unneeded you can safely ignore this parameter.  
```

Comment:
```suggestion
    > Because Visual Studio Code source path mapping requires the URL, your application now receives a command line parameter when it starts. You can safely ignore this parameter if needed.  
```